python/cudf/cudf/_lib/sort.pyx

```diff
@@ -153,6 +162,11 @@ def order_by(list columns_from_table, object ascending, str na_position):
                 True - Ascending and False - Descending
     na_position : whether null value should show up at the "first" or "last"
                 position of **all** sorted column.
+    stable : should the sort be stable?
```

Comment:
We're not using proper numpydoc parameter formatting here... https://numpydoc.readthedocs.io/en/latest/format.html#parameters

But it's okay with me if that change is considered out-of-scope for this PR. 😕